ndex2/client.py

```diff
@@ -749,8 +1025,19 @@ def delete_network(self, network_id, retry=5):
 
     def get_provenance(self, network_id):
         """
+
         Gets the network provenance
 
+        .. code-block:: python
+         
+		import ndex2.client as nc
+		import ndex2
+		import networkx as nx
+		import pandas as pd
+		import os
+
+            my_ndex.get_provenance('INSERT NETWORK UUID HERE') 
```

Comment:
Please add line showing how `my_ndex` is created. The only import needed is the one to import the Ndex2 client